### ISSUE TITLE
Scenario Bundles: model / framwork dropdown with abbreviations instead

### ISSUE THREAD

**Issue by han-f** [openego/oeplatform#1781]
## Description of the issue

I think for users of the scenario bundles it is easier to find models in a dropdown via the abbreviation than the full name of the model. Thus I want to suggest to query the abbreviations of models instead of their full names in the scenario bundles. 

## Ideas of solution

Describe possible ideas for solution and evaluate advantages and disadvantages.

## Workflow checklist

- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md)
 cc @l-emele @stap-m @wingechr

**Comment by jh-RLI** [comment]
I agree, I have used the name because it is a mandatory field when users create a model fact sheet. Otherwise, only the user's contact email is mandatory. The field for the acronym is not mandatory and can therefore be empty. 

I could do it here again as I did for the code that displays the name or acronym if it's available. 

I give it an urgency level m as the functionality is there but could be improved. 

Translated with DeepL.com (free version)

**Comment by han-f** [comment]
Thanks. On the end of the model factsheets: Might it be a possibility to automatically fill the abbreviation field in the model factsheets with the name in case a user does not provide an abbreviation?

**Comment by jh-RLI** [comment]
I will try it this way. We can also make this field mandatory. But then we have to edit all factsheets where this field is empty. I assume that would take a bit of time.
